datasource/ethereum/src/ethereum_adapter.rs

```diff
@@ -97,6 +171,40 @@ where
             })
     }
 
+    fn trace_stream(
+        self,
+        logger: &Logger,
+        from: u64,
+        to: u64,
+        addresses: Vec<H160>,
+    ) -> impl Stream<Item = Vec<Trace>, Error = Error> + Send {
+        if from > to {
+            panic!(
+                "Can not produce a call stream on a backwards block range: from = {}, to = {}",
+                from, to,
+            );
+        }
+
+        let eth = self.clone();
+        let logger = logger.to_owned();
+        stream::unfold(from, move |start| {
+            if start > to {
+                return None;
+            }
+            let end = (start + 200 - 1).min(to);
```

Comment:
This could use a comment — what does it mean that we pick 200 blocks here and why?

Reply:
Made this controllable by env var, still defaults to 200 though

Reply:
That's alright